### ISSUE TITLE
Compiler warning on Linux due to unused result (-Wunused-result)

### ISSUE THREAD

**Issue by null77** [KhronosGroup/Vulkan-ValidationLayers#2430]
Error showed up here:

https://chromium-review.googlesource.com/c/angle/angle/+/2601340

https://ci.chromium.org/ui/p/angle/builders/try/linux-clang-rel/18555/overview

```
../../third_party/vulkan-deps/vulkan-loader/src/loader/loader.c:2659:9: error: ignoring return value of function declared with 'warn_unused_result' attribute [-Werror,-Wunused-result]
        fread(buffer, 1, sizeof(buffer), file);
        ^~~~~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

@mark-lunarg is this warning enabled in the loader build? Should we fix the specific warning and/or enable it in the CI or should I remove this warning from the supported set in GN?

**Comment by null77** [comment]
Whoops, wrong repo.
